project-fortis-services/src/clients/appinsights/LoggingClient.js

```diff
@@ -35,6 +35,39 @@ function restartPipelineExtraProps() {
   });
 }
 
+function usersExtraProps() {
+  return () => ({
+    operation: 'query',
+    table: 'users',
+    success: 'true'
+  });
+}
+
+function addUsersExtraProps() {
+  return () => ({
+    operation: 'modify',
+    table: 'users',
+    success: 'true'
+  });
+}
+
+function removeUsersExtraProps() {
+  return () => ({
+    operation: 'remove',
+    table: 'users',
+    success: 'true'
+  });
+}
+
+function usersExtraMetrics() {
+  return (graphqlResult) => {
+    const totalRows = graphqlResult.users.length;
```

Comment:
Please make this error-proof by protecting against potential null values of graphqlResult or users. This happened in at least one case so far (see https://github.com/CatalystCode/project-fortis/commit/182e2dfb52a53b5b635ef86d3fa1de86e31dd4eb).